scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -3,10 +3,20 @@ set -e -o verbose
 
 export DEBIAN_FRONTEND=noninteractive
 
+apt update --yes
```

Comment:
How about we just update the packages we care about and leave the rest?

Reply:
I feel like its a) good hygiene and b) there's no guarantee this won't be triggered by a future package install if a dependency changes.